src/STEPToMesh.cpp

```diff
@@ -138,14 +145,15 @@ int main(int argc, char* argv[]) {
 			("s,select", "Select solids by name or index (comma seperated list, index starts with 1)", cxxopts::value<std::vector<std::string>>())
 			("l,linear", "Linear deflection", cxxopts::value<double>())
 			("a,angular", "Angular deflection (degrees)", cxxopts::value<double>())
+			("u,unit", "Output unit (default mm)", cxxopts::value<std::string>()->default_value("MM"))

```

Comment:
Should we list the supported unit of OpenCASCADE here? (does cxopts support multiline help texts?)

Reply:
multiline help texts (with newlines) are currently not supported (would need an update of cxxopts, see https://github.com/jarro2783/cxxopts/pull/215), but I have added the allowed units anyway (comma separated)